nautobot/ipam/views.py

```diff
@@ -753,6 +753,8 @@ def get_extra_context(self, request, instance):
             .filter(group=instance)
             .prefetch_related(Prefetch("prefixes", queryset=Prefix.objects.restrict(request.user)))
         )
+        
```

Comment:
One change needed to satisfy Black format checking - remove trailing whitespace on this line:

```suggestion

```